src/main/resources/liquibase/workbench_changelog/15_4_0.xml

```diff
@@ -22,4 +22,24 @@
 		</sql>
     </changeSet>
 
+	<changeSet author="aldrin" id="v15.4.0-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM permission where name = 'MS_CREATE_LOTS'
+			</sqlCheck>
+        </preConditions>
+        <comment>Create granular permissions for Manage studies: Create Lots</comment>
+		<sql dbms="mysql" splitStatements="true">
+			set @manage_studies_id = (select permission_id from permission where name = 'MANAGE_STUDIES');
+			set @sidebar_category_link_id = (select sidebar_category_link_id from workbench_sidebar_category_link where tool_name = 'study_manager_fieldbook_web');
+
+			insert into permission(name, description, parent_id, workbench_sidebar_category_link_id)
```

Comment:
Can't we just have null values for `workbench_sidebar_category_link_id`?

Reply:
I populated this for consistency with other permission records.